### ISSUE TITLE
The `<svg>` acts as an image with a text alternative, missing an explicit `role="img"`.

### ISSUE THREAD

**Issue by asthaawasthi-laitkor** [dequelabs/axe-core#4308]
### Product

axe-core

### Question

i am using axe-core api  to to check error ,failures with WCAG SC
,for the attached source code I were the expecting an  issue in the results:: `<svg>` acts as an image with a text alternative, but is missing an explicit `role="img"`. or somthing like this ,but didn't receive any issue with <svg> tag , please explain the reason for this..  
source code 

```html
<a class="footer-nav-link" href="#" data-modal="open" data-cache="true" data-options="{}" data-tracking="{}" title="Pet Policy" aria-label="click to view pet policy" aria-haspopup="dialog">
  <svg viewBox="0 0 32 32" class="pp-svg-icon icon paw-icon">
    <use xlink:href="/Common/website_templates/_common/fonts/svg_sprite.svg#paw-icon"></use>
  </svg>
</a>
```

**Comment by straker** [comment]
Thanks for the question. Axe-core does not have any rules to detect when elements that should have aria roles don't have them. Axe-core can only check when roles are used incorrectly. These type of cases can only be caught by a manual accessibility review.

**Comment by asthaawasthi-laitkor** [comment]
Thank you @straker for the information
